<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>961</width>
      <height>661</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;文件</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_as"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Paste"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;查找</string>
    </property>
    <addaction name="action_Find"/>
   </widget>
   <widget class="QMenu" name="menu_Format">
    <property name="title">
     <string>&amp;格式</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionColor"/>
    <addaction name="actionGreen"/>
    <addaction name="actionBlue"/>
    <addaction name="actionRed"/>
    <addaction name="actionBlack"/>
    <addaction name="separator"/>
    <addaction name="actionBackground_color"/>
    <addaction name="actionBlack_BG"/>
    <addaction name="actionWhite_BG"/>
    <addaction name="separator"/>
    <addaction name="actionFont_Size"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;帮助</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_About_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_History">
    <property name="title">
     <string>&amp;历史</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
   <addaction name="menu_Format"/>
   <addaction name="menu_About"/>
   <addaction name="menu_History"/>
  </widget>
  <widget class="QToolBar" name="EdittoolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Cut"/>
   <addaction name="action_Paste"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Find"/>
   <addaction name="actionGreen"/>
   <addaction name="actionBlue"/>
   <addaction name="actionRed"/>
   <addaction name="actionBlack"/>
   <addaction name="actionBlack_BG"/>
   <addaction name="actionWhite_BG"/>
  </widget>
  <widget class="QToolBar" name="FiletoolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Save_as"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/new.png</normaloff>:/image/image/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/open.png</normaloff>:/image/image/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/save.png</normaloff>:/image/image/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/save.png</normaloff>:/image/image/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;另存为</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/exit.png</normaloff>:/image/image/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;退出</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/copy.png</normaloff>:/image/image/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/cut.png</normaloff>:/image/image/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/paste.png</normaloff>:/image/image/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/find.png</normaloff>:/image/image/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;查找</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/green.png</normaloff>:/image/image/green.png</iconset>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/blue.png</normaloff>:/image/image/blue.png</iconset>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/red.png</normaloff>:/image/image/red.png</iconset>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/black.png</normaloff>:/image/image/black.png</iconset>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionBackground_color">
   <property name="text">
    <string>Background color</string>
   </property>
  </action>
  <action name="actionGreen_BG">
   <property name="text">
    <string>Green_BG</string>
   </property>
  </action>
  <action name="actionBlue_BG">
   <property name="text">
    <string>Blue_BG</string>
   </property>
  </action>
  <action name="actionBlack_BG">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/1.png</normaloff>:/image/image/1.png</iconset>
   </property>
   <property name="text">
    <string>Black_BG</string>
   </property>
  </action>
  <action name="actionWhite_BG">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/2.png</normaloff>:/image/image/2.png</iconset>
   </property>
   <property name="text">
    <string>White_BG</string>
   </property>
  </action>
  <action name="actionFont_Size">
   <property name="text">
    <string>Font Size</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Copy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Cut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Paste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
